.scalafmt.conf

```diff
@@ -1,5 +1,5 @@
 version = "3.7.15"
-runner.dialect = scala212
+runner.dialect = scala3
```

Comment:
I commited this by accident but.. since this is a Scala 3 project only, was the Scala212 choice accidental?

Reply:
Probably was accidental, not sure if you need special settings for Scala212 on SBT files.
Anyways, if we change this we should also remove the `fileOverride` below.

Reply:
Cleaned up in [1f2fbcc](https://github.com/lolgab/scala-native-jdbc/pull/5/commits/1f2fbccbdec15a062ca7f69bf1be000a0457f719)